2024_homework/add_up2.rb

```diff
@@ -0,0 +1,18 @@
+# (3) Write a program with a function add_up(i) .  It is to be passed a positive integer, and it will add all the numbers from 1 to that integer and return the sum.  Call the function three times within the program, and each time print out the return value.  Call the program add_up.rb.
+
+def add_up(i)
```

Comment:
The function works fine! I'd just consider increasing the readability of the code with more descriptive variable names.